src/cli.rs

```diff
@@ -148,6 +149,7 @@ fn str_to_backup_folder(s: &str) -> Result<BackupFolder, &'static str> {
         "wtf" => Ok(BackupFolder::Wtf),
         "addons" => Ok(BackupFolder::AddOns),
         "config" => Ok(BackupFolder::Config),
-        _ => Err("valid values are ['all','wtf','addons','config']"),
+        "screenshots" => Ok(BackupFolder::Screenshots),
+        _ => Err("valid values are ['all','wtf','addons','config', 'screenshots']"),
```

Comment:
Can you also add `screenshots` to the structopt description for field `backup_folder`... Line 102

Reply:
This is what prints when an error occurs (user passes the wrong text). The description from line 102 is what prints on `--help`

Reply:
Added!